templates/vue/src/components/TapestryApp.vue

```diff
@@ -41,6 +41,7 @@
             @dragend="updateViewBox"
             @mouseover="handleMouseover(id)"
             @mouseleave="activeNode = null"
+            @nodemounted="updateSelectableNodes"
```

Comment:
Could we perhaps rename this event to `mounted`? I think it reads a bit better.

```suggestion
            @mounted="updateSelectableNodes"
```